src/components/common/dialogs/CustomDialog.js

```diff
@@ -73,4 +108,41 @@ const SimpleStoreDialog = () => {
   )
 }
 
+const styles = StyleSheet.create({
```

Comment:
we can use withStyles if we need to load theme variables from StyleSheet, I think it is better than importing the theme